### PR TITLE
osgeo-incubation, Added Copyright Headers to all python files

### PR DESCRIPTION
**Pull request by BerryDaniel**
#2348 GeoNode has been signed over from Open Plans to OSGeo.

### CODE REVIEW

**Comment by tomkralidis** [comment]
cc @bitner

@BerryDaniel great! Should the copyright read `OSGeo` or `Open Source Geospatial Foundation` or `Open Source Geospatial Foundation (OSGeo)`?  This is a minor detail, but worth clarifying.

**Comment by bitner** [comment]
I'm not sure that it really matters, I think either are perfectly clear as to the intent. I'd be inclined to leave as is unless during final review it is brought up by someone on the incubation committee.

**Comment by tomkralidis** [comment]
@bitner fair enough, just considering the [GeoServer](https://github.com/geoserver/geoserver/blob/master/src/ows/src/main/java/org/geoserver/ows/AbstractDispatcherCallback.java) case as a similar scenario/lesson learned.

**Comment by jodygarnett** [comment]
You can see examples here [here](http://docs.geoserver.org/stable/en/developer/policies/committing.html#commit-guidelines).

Because this is donating the code should be a search and replace for OpenPlans.

Should be something like BEFORE:
````
#########################################################################
#
# Copyright (C) 2012 OpenPlans
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program. If not, see <http://www.gnu.org/licenses/>.
#
#########################################################################
````

AFTER:
````
#########################################################################
#
# Copyright (C) 2012 OSGeo
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program. If not, see <http://www.gnu.org/licenses/>.
#
#########################################################################
````

File: geonode/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# -*- coding: utf-8 -*-
 #########################################################################
 #
-# Copyright (C) 2012 OpenPlans
+# Copyright (C) 2016 OSGeo
```
Comment (jodygarnett) [inline geonode/__init__.py]: This is a code donation, the above should be:
````
# Copyright (C) 2012 Open Source Geospatial Foundation
````

Unless the file was modified in 2016:
````
# Copyright (C) 2012-2016 Open Source Geospatial Foundation
````

**Pull request merged**